jaas/src/main/java/com/griddynamics/jagger/jaas/JaasRestController.java

```diff
@@ -0,0 +1,99 @@
+package com.griddynamics.jagger.jaas;
+
+import com.griddynamics.jagger.engine.e1.services.DataService;
+import com.griddynamics.jagger.engine.e1.services.data.service.MetricEntity;
+import com.griddynamics.jagger.engine.e1.services.data.service.MetricPlotPointEntity;
+import com.griddynamics.jagger.engine.e1.services.data.service.MetricSummaryValueEntity;
+import com.griddynamics.jagger.engine.e1.services.data.service.SessionEntity;
+import com.griddynamics.jagger.engine.e1.services.data.service.TestEntity;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * JaaS REST API controller based on Spring MVC.
+ */
+@RequestMapping(value = "/jaas")
+@RestController
+public class JaasRestController {
+
+    private final static Logger LOGGER = LoggerFactory.getLogger(JaasRestController.class);
+
+    @Autowired
+    private DataService dataService;
+
+    @GetMapping(value = "/sessions/{sessionId}")
+    SessionEntity getSession(@PathVariable String sessionId) {
+        LOGGER.debug("Input session id: {}", sessionId);
+        return dataService.getSession(sessionId);
+    }
+
+    @GetMapping(value = "/sessions")
+    Set<SessionEntity> getSessions(@RequestParam(name = "id", required = false) String[] sessionIds) {
+        LOGGER.debug("Input session ids: {}", sessionIds);
+        if (sessionIds == null) {
+            sessionIds = new String[0];
+        }
+        return dataService.getSessions(Arrays.asList(sessionIds));
+    }
+
+    @GetMapping(value = "/sessions/{sessionId}/tests/{testName}")
+    TestEntity getTest(@PathVariable String sessionId, @PathVariable String testName) {
+        LOGGER.debug("Input session id: {}", sessionId);
+        LOGGER.debug("Input test name: {}", testName);
```

Comment:
Let me share my thoughts about it.
1. I am not sure how informative will be those events. (this is another point though)
    i.e. 
    Tue Aug 30 17:35:51.096 PDT 2016| DEBUG | Input session id: 32
    Tue Aug 30 17:35:51.097 PDT 2016| DEBUG | Input test name: testName
    Tue Aug 30 17:35:54.096 PDT 2016| DEBUG | Input session id: 32
    Tue Aug 30 17:35:56.096 PDT 2016| DEBUG | Input session id: 32
    Tue Aug 30 17:35:51.097 PDT 2016| DEBUG | Input test id: 545
2. 2 log events will be handled one by one. Of course additional log event will not make any impact on performance/IO etc., but why to do one thing twice.
**3.** As I can see Jagger starting from this commit is going in to the direction of an Application as a service. That means that in future there will appear new task about monitoring the logs on regular basis. It is simply easier to extract all necessary information about single event (executing JaasRestControkller.getTest method) from one log event.
4. As I understand information of the executing different REST methods will be available in access log :)